src/LightQuery.Shared/QueryableProcessor.cs

```diff
@@ -7,6 +9,43 @@ namespace LightQuery.Shared
 {
     public static class QueryableProcessor
     {
+        public static PropertyInfo GetPropertyInfoRecursivly(this IQueryable queryable, String propName)
```

Comment:
The name should be "**GetPropertyInfoRecursively**"

Reply:
Fixed it 👍